python/lbann/launcher/lsf.py

```diff
@@ -53,47 +53,36 @@ def __init__(self,
                          interpreter=interpreter)
         self.nodes = nodes
         self.procs_per_node = procs_per_node
+        self.reservation = reservation
         self.launcher = launcher
         self.launcher_args = launcher_args
 
         # Configure header with LSF job options
-        self._construct_header(job_name=job_name,
-                               nodes=self.nodes,
-                               time_limit=time_limit,
-                               partition=partition,
-                               account=account,
-                               reservation=reservation)
-
-    def _construct_header(self,
-                          job_name=None,
-                          nodes=1,
-                          time_limit=None,
-                          partition=None,
-                          account=None,
-                          reservation=None):
-        """Construct script header with options for bsub."""
+        self.add_header_line(f'#BSUB -cwd {self.work_dir}')
```

Comment:
Header lines are only needed if submitting a batch job not in interactive (default) mode right? I don't see where add_header  is optional for both LSF and SLURM

Reply:
Actually, I see this is only when batch scripts are constructed, so ignore.